common/transport/amqp/package.json

```diff
@@ -25,12 +24,19 @@
     "istanbul": "^0.4.5",
     "mocha": "^5.2.0",
     "sinon": "^4.0.2",
+    "sinon-test": "^2.2.1",
     "tslint": "^5.1.0",
     "typescript": "2.5.2"
   },
   "scripts": {
     "lint": "tslint --project . -c ../../../tslint.json",
     "build": "tsc",
+    "all_together": "istanbul cover node_modules/mocha/bin/_mocha -- --reporter spec test/_amqp_cbs_test.js test/_amqp_common_errors_test.js test/_amqp_link_test.js test/_amqp_message_test.js test/_sender_link_test.js test/_receiver_link_test.js",
+    "sender_link": "istanbul cover node_modules/mocha/bin/_mocha -- --reporter spec test/_sender_link_test.js",
```

Comment:
use a glob pattern instead? test/_*_test.js?

Reply:
Actually all of these can be removed.  Just was adding them as I was going through the unit test exercise.

---
In reply to: [209123651](https://github.com/Azure/azure-iot-sdk-node/pull/344#discussion_r209123651) [](ancestors = 209123651)